code/datums/shuttle.dm

```diff
@@ -503,7 +503,7 @@
 
 	var/area/space
 
-	space = get_space_area
+	space = get_space_area()
 	if(!space)
 		warning("There is no area at 1,1,2!")
```

Comment:
1,1,2 was used because there was no better way to find the space area object -- locate(/area) returned the wrong one. this PR makes it so that it's no longer used

and the landmark is used for dungeons #11282